cf_mssql_broker_config.json

```diff
@@ -32,7 +32,7 @@
 	"brokerMssqlConnection": {
 		"server":   "localhost\\sqlexpress",
 		"database": "master",
-		"driver": 	"sql server",
+		"driver": 	"{SQL Server Native Client 11.0}",
```

Comment:
We should not use the old driver is, you should use the latest ODBC driver, the question is how you get it on the node as it is not installed by default on a Windows node. It will be there if SQL Server is installed, otherwise you have to download Microsoft® SQL Server® 2012 Native Client pack http://go.microsoft.com/fwlink/?LinkID=239648&clcid=0x409